lib/page/profile.dart

```diff
@@ -209,15 +212,20 @@ class _ProfileState extends State<Profile> {
           height: 7.0,
         ),
         Wrap(
-            spacing: 10.0,
-            runSpacing: 16.0,
-            children: buildInterest(user.preferences)),
+          spacing: 8.0,
+          runSpacing: 0.0,
+          children: buildInterest(user.preferences),
+        ),
         const SizedBox(height: 34.0),
         CustomButton(
           label: "Edit Profile",
           type: ButtonType.primary,
-          onPressedHandler: () {
-            Navigator.pushNamed(context, EditProfile.routeName);
+          onPressedHandler: () async {
+            UserModel updatedUser =
+                await Navigator.pushNamed(context, EditProfile.routeName)
+                    as UserModel;
+            passedUser = updatedUser;
```

Comment:
why is setState empty? I know it'll still trigger the change but I think it's better to put variables that wants to be updated inside setState